bin/Powheg/production/V2/13TeV/Higgs/HJJ_JHUGen_HZZ4L_NNPDF30_13TeV/HJJ_NNPDF30_13TeV_M300.input

```diff
@@ -0,0 +1,92 @@
+numevts NEVENTS     ! number of events to be generated
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500     ! energy of beam 1
+ebeam2 6500     ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1   260000      ! pdf set for hadron 1 (LHA numbering)
+lhans2   260000      ! pdf set for hadron 2 (LHA numbering)
+! To be set only if using internal mlm pdf
+#ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+#ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets 
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1 ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+
+hmass  300             ! Higgs mass
+hwidth 8.43D0            ! Higgs width
+
+bwcutoff 15d0       ! Mass window is hmass +- bwcutoff*hwidth
+
+#higgsfixedwidth 1  ! (default 0), If 1 uses standard, fixed width Breit-Wigner
+                   ! formula, if 0 it uses the running width Breit-Wigner
+
+#ckkwscalup 1 ! (default 1), If 1 compute the scalup scale for subsequent
+              ! shower using the smallest kt in the final state;
+              ! If 0, use the standard POWHEG BOX scalup
+
+#runningscales 1   ! (default 0), if 0 use hmass as central
+                  ! factorization and renormalization scale;
+                  ! if 1 use the Ht scale
+
+# fullphsp 1         ! use ISR/FSR phase space parametrization, default 0, do not
+
+
+ncall1   150000   ! number of calls for initializing the integration grid
+itmx1    3      ! number of iterations for initializing the integration grid
+ncall2   100000     ! number of calls for computing the integral and finding upper bound
+itmx2    3  ! number of iterations for computing the integral and finding upper bound
+foldcsi   2    ! number of folds on csi integration
+foldy     5     ! number of folds on  y  integration
+foldphi   2      ! number of folds on phi integration
+nubound  60000    ! number of bbarra calls to setup norm of upper bounding function
```

Comment:
nubound seems to be a bit on the low side (see comments on the powheg twiki) have you done tests to ensure that the distributions are smooth?